README.MD

```diff
@@ -36,7 +36,7 @@ You can reference the following Jobs provided by this orb from a 2.1 Workflows c
 | cluster-name   | string  |   | The short name or full ARN of the cluster that hosts the service |
 | service-name   | string  |   | The name of the service to update. If undefined, we assume `family` is the name of both the service and task definition. |
 | container-image-name-updates      | string   | ""    | Use this to update the Docker image names and/or tag names of existing containers that had been defined in the previous task definition. <br>**Expected format**: container=\<container-name\>,image-and-tag=\<image-name\>:\<tag-name\>\|image=\<image-name\>\|tag=\<tag-name\>,container=...,image-and-tag\|image\|tag=... <br>For each container, specify only either `image-and-tag` or `image` or `tag`.<br>If `image-and-tag` is specified, the container image will be updated to the value of the name-value pair.<br>If `image` is specified, the image tag defined in the previous task definition will be retained, if exists.<br>If `tag` is specified, the image name defined in the previous task definition will be used. |
-| container-env-var-updates        | string  | ""   | "Use this to update or set the values of environment variables that will be defined for the containers. (Existing environment variables not included in this parameter will not be removed) <br>**Expected format**: container=\<container-name\>,name=\<env-var-name\>,value=\<env-var-value\>,container=...,name=...,value=...,<br>Values should not contain commas. |
+| container-env-var-updates        | string  | ""   | "Use this to update or set the values of environment variables that will be defined for the containers. (Existing environment variables not included in this parameter will not be removed) <br>**Expected format**: container=\<container-name\>,name=\<env-var-name\>,value=\<env-var-value\>,container=...,name=...,value=...,container=...,name=...,valueFrom=...,<br>Values should not contain commas. |
```

Comment:
Thanks for updating the descriptions here! We would also want to update the descriptions in `orb.yml.hbs` so that the orb's source is also updated with the descriptions.